js/text-insertion.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+let textToInsert, dxOfText, dyOfText;
+
+let textMenu = document.getElementById('textMenu');
+let pastedText = document.getElementById('textIndicator');
+let textFormat = document.getElementById('chooseTextFormat');
+let fontSize = document.getElementById('textSize');
+let fontColor = document.getElementById('textColor');
+let font = document.getElementById('fontSelector');
+let textDeg = document.getElementById('textAngle');
+
+function chooseTextFormat() {
+  writeText(canvas.width / 2 - dxOfText, canvas.height / 2 - dyOfText);
+}
+
+function initText() {
+  saveImg();
+  pastedText.hidden = false;
+  pastedText.style.zIndex = activeLayer.index + 1 + '';
+  document.addEventListener('keydown', pressForInsertion);
+
+  function pressForInsertion() {
+    if (event.code == 'Enter' && event.altKey) {
+      dxOfText = pastedText.clientWidth;
+      dyOfText = pastedText.clientHeight;
+      pastedText.hidden = true;
+      textMenu.hidden = false;
+      textMenu.style.zIndex = activeLayer.index + 1 + '';
```

Comment:
```suggestion
      textMenu.style.zIndex = activeLayer.index + 1;
```
Все автоматом приведется к строке

Reply:
Ок